scrapy_zyte_api/addon.py

```diff
@@ -0,0 +1,24 @@
+from scrapy.settings import BaseSettings
+
+from scrapy_zyte_api import ScrapyZyteAPIDownloaderMiddleware
+
+
+class ScrapyZyteAPIAddon:
+    def update_settings(self, settings: BaseSettings) -> None:
+        settings["DOWNLOAD_HANDLERS"][
+            "http"
+        ] = "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler"
+        settings["DOWNLOAD_HANDLERS"][
+            "https"
+        ] = "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler"
```

Comment:
Should we add some checks for non-standard http/https handlers?

Reply:
How about this?

I don't like this approach because it assumes the default values. We could import `scrapy.settings.default_settings` instead (hopefully no settings modify the mutable objects there), or for `DOWNLOAD_HANDLERS` just check its value without `getwithbase()`.